docs/03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md

```diff
@@ -21,17 +21,19 @@ To subscribe to multiple events, you need a [Subscription](../../05-technical-re
   Kyma Dashboard
   </summary>
 
-1. In Kyma Dashboard, go to the view of your Function `lastorder`.
-2. Go to **Configuration** > **Create Subscription+**.
-3. Switch to the **Advanced** tab, and name the Subscription `lastorder-sub`.
-4. Add a second Filter using **Filters** > **Add Filter +**.
-5. Provide the `Event type` under `Filter 1` as `sap.kyma.custom.myapp.order.received.v1`. Leave the `Event source` as empty.
-6. Provide the `Event type` under `Filter 2` as `sap.kyma.custom.myapp.order.changed.v1`. Leave the `Event source` as empty.
+1. Go to **Namespaces** and select the default Namespace.
+2. Go to **Configuration** > **Subscriptions** and click **Create Subscription+**.
+3. Provide the following parameters:
```

Comment:
(nit)
here again, could we change the sequence ?

Reply:
updated at all places.